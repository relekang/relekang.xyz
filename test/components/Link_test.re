open Jest;

open ExpectJs;

describe
  "<Link />"
  (
    fun _ =>
      test
        "render matches snapshot"
        (
          fun _ =>
            <Link url="http://relekang.xyz" title="o/" /> |> ReactTestRenderer.create |> expect |> toMatchSnapshot
        )
  );
