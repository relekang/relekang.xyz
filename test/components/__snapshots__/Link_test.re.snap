// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Link /> render matches snapshot 1`] = `
<a
  className={
    Object {
      "data-css-hh2ow": "",
    }
  }
  href="http://relekang.xyz"
>
  o/
</a>
`;
